api/types/container/container_top.go

```diff
@@ -0,0 +1,21 @@
+package container
+
+// ----------------------------------------------------------------------------
+// DO NOT EDIT THIS FILE
+// This file was generated by `swagger generate operation`
+//
+// See hack/swagger-gen.sh
+// ----------------------------------------------------------------------------
+
+// ContainerTopOKBody container top o k body
```

Comment:
this `top o k body` seems wrong (reminds me ok `g ID`)

Even the d name feels weird no ? `OK` in the variable.